policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/child_assistance/back_to_school_amount.yaml

```diff
@@ -0,0 +1,9 @@
+description: Quebec Child Assistance Tax Credit back to school amount.
```

Comment:
Change description to:
Quebec provides this amount to filers with children who are eligible under the age threshold.